### DISCUSSION TITLE
Contact Form Submission - Suggestion

### DISCUSSION THREAD

**Discussion by maggieliu05** [cpinitiative/usaco-guide#2691] [Ideas]
Someone submitted the contact form!

**Module**: > Silver
**Topic**: Suggestion
**Message**: 
Heyy... A fantastic day to everyone at USACO... Thank you for putting together this wonderful and detailed resource. 

I do a suggestion, which is that the people submitting editorial solutions should include a short description of the intuition behind the problem... That way, folks necessarily don't have to go through all of the code to be able to also figure it out and implement it on their own. 

The nice thing about this, which builds on the nice thing about USACO Guide's crowdsourcing editorial is that people will easily identify which intuition it is that they were already building and figure out where they made mistakes or errors in their train of logic, and work on that. 

Thank you!
Stay Stellar,
Israel

**Comment by SansPapyrus683** [comment]
so what is this saying now?
that we include a tldr with every editorial?
